Gemfile.lock

```diff
@@ -30,23 +30,23 @@ GEM
     jmespath (1.4.0)
     mandate (0.3.0)
     minitest (5.14.1)
-    parallel (1.19.2)
-    parser (2.7.1.4)
+    parallel (1.20.0)
+    parser (2.7.2.0)
       ast (~> 2.4.1)
     rainbow (3.0.0)
-    regexp_parser (1.7.1)
+    regexp_parser (1.8.2)
     rexml (3.2.4)
-    rubocop (0.89.1)
+    rubocop (1.3.1)
```

Comment:
I've updated Rubocop to a more recent version, as I thought this was what caused the build to fail.